consensus/src/payload_manager.rs

```diff
@@ -604,11 +643,13 @@ async fn process_payload_helper<T: TDataInfo>(
                 .batch_summary
                 .iter()
                 .map(|proof| {
-                    let mut signers = proof.signers(ordered_authors);
+                    let mut signers = signers.clone();
+                    signers.append(&mut proof.signers(ordered_authors));
```

Comment:
we add the qc signers to proof responders too?

Reply:
it's only passed for opt batches not proofs. `proof` is a data ptr. will rename.